### ISSUE TITLE
[Feature Request] Jewels passives transformation.

### ISSUE THREAD

**Issue by krab** [Openarl/PathOfBuilding#44]
http://i.imgur.com/SMLO8zW.jpg

It's better to add section with new modifiers descriptions under original. Something like this:

    30% increased Physical Damage with Maces
    40% increased Stun Duration on Enemies
    5% increased Radius of Area Skills
    ----------------------------------------------
    30% increased Physical Damage with Bows
    40% increased Stun Duration on Enemies
    5% increased Radius of Area Skills

**Comment by Openarl** [comment]
That is actually quite difficult to do, due to the way the program handles jewel conversions; it translates the user-readable modifier description into the program's internal modifier format before applying the conversion, and there's no easy way of translating the new modifier back into text.

One solution I've considered is displaying a modifier table in each passive node tooltip (similar to the ones that you see in the Calcs tab). It'd still require a fair bit of work to implement though.

**Comment by krab** [comment]
Yeah, i mean add new modifiers group under original, maybe with different
color. 🤔

24 нояб. 2016 г. 9:03 AM пользователь "Openarl" <notifications@github.com>
написал:

**Comment by myrahz** [comment]
Instead of converting the mods to bow, wouldn't it be easier to just consider the nodes inside the jewel radius get applied to the bow? Similar to Varunastra.

**Comment by Openarl** [comment]
This issue isn't about converting the actual modifiers, but rather changing the modifier descriptions in the node tooltip to match the converted modifiers. My stance on that hasn't changed: would be nice, but a MAJOR pain to do.
